README.md

```diff
@@ -1,33 +1,45 @@
-# Flight HUD
+<p align=center>
+    <img src="images/logo.png">
+</p>
 
-__Flight HUD__ is a client side Fabric Minecraft Mod that adds a flight style HUD like one you would see in a flight
-simulator.
+<p align=center>
+    <a href="https://modrinth.com/mod/flightassistant">
+        <img src="https://cdn.jsdelivr.net/npm/@intergrav/devins-badges@3.1.2/assets/cozy/available/modrinth_vector.svg"
+            alt="Available on Modrinth"></img></a>
+    <a href="https://modrinth.com/mod/fabric-api/">
+        <img src="https://cdn.jsdelivr.net/npm/@intergrav/devins-badges@3/assets/cozy/requires/fabric-api_vector.svg"
+            alt="Requires Fabric API"></img></a>
+    <a href=""> <!-- don't open image preview -->
+        <img src="https://cdn.jsdelivr.net/npm/@intergrav/devins-badges@3.1.2/assets/cozy/unsupported/forge_vector.svg"
+            alt="Won't support Forge"></img></a>
+</p>
 
-![HUD Screenshot](images/hud2020-07-07.png?raw=true "HUD Screenshot")
+---
+
+**FlightAssistant** is a client side Fabric Minecraft Mod that adds a flight style HUD like one you would see in a flight simulator with some features.
 
 ## Features
 
-- pitch ladder
-- artificial horizon
-- flight path (prograde vector)
-- air speed with sliding scale
-- heading with sliding scale
-- direction ordinals and axes
-- altitude with sliding scale (relative to block zero)
-- height (distance from ground)
-- elevation summary showing
-    - void damage mark
-    - block zero
-    - ground level
-    - build limit
-    - current altitude
-- elytra health percentage
-- location in x/z
-
-![HUD Components](images/hud-diagram.png?raw=true "HUD Components")
-
-## Requirements
-
-- Minecraft 1.19.4
-- Fabric Mod Loader
-- Fabric API Mod
```

Comment:
why was this deleted?

Reply:
The Fabric API is already mentioned in the badge, and supported versions of Minecraft will be available on the downloads page on Modrinth.